Client/Frontend/Home/JumpBackIn/JumpBackInViewModel.swift

```diff
@@ -394,14 +398,17 @@ extension JumpBackInViewModel: HomepageViewModelProtocol {
                      size: CGSize,
                      isPortrait: Bool = UIWindow.isPortrait,
                      device: UIUserInterfaceIdiom = UIDevice.current.userInterfaceIdiom) {
-        updateSectionLayout(for: traitCollection,
-                            isPortrait: isPortrait,
-                            device: device)
         let maxItemsToDisplay = sectionLayout.maxItemsToDisplay(
             hasAccount: isSyncTabFeatureEnabled,
             device: device
         )
         refreshData(maxItemsToDisplay: maxItemsToDisplay)
+        logger.log("JumpBackIn section shouldShow \(shouldShow)",
+                   level: .debug,
+                   category: .homepage)
+        updateSectionLayout(for: traitCollection,
+                            isPortrait: isPortrait,
+                            device: device)
```

Comment:
I remember the order for the `updateSectionLayout` was finicky. We want to make this change now?

Reply:
Not necessarily I can revert this part until we have more info, the reasoning behind the change was to update the dataSource before updating the layout the little information I found around the crash mentioned that dataSource changes should be done before updating the layout